kubernetes/apps/manifests/gtfs-rt-archiver-v3/redis.yaml

```diff
@@ -41,6 +44,22 @@ spec:
               cpu: 1
             limits:
               memory: 1Gi
+        # from https://github.com/oliver006/redis_exporter
+        - name: redis-exporter
+          image: oliver006/redis_exporter:latest
```

Comment:
Should we consider forking this for stability? It would introduce manual update burdens, but minimize risks from malicious takeover of an individually maintained repo

Reply:
I think it's a valid concern but yeah does have to be balanced against maintenance as well as the risk of existing issues that get patched but we fail to update regularly. I'll pin to v1.46.0 as a middle-ground.